src/styles/blocks/_calendar.scss

```
@@ -0,0 +1,21 @@
+.calendar {
+  padding: 40px 0;
+  max-width: $calendar-width;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $days-gap;
+}
+
+.calendar__days {
+  width: $days-size;
+  height: $days-size;
+  position: relative;
+  border: 1px solid black;
+  background: #eee;
+
+  &:hover {
+    background: #ff9898;
+    transform: translate(0, -20%);
+    transition: 0.5s;
+  }
+}
```

you can use the nesting here
```suggestion
  gap: $days-gap;
  
  &__days {
    width: $days-size;
    height: $days-size;
    position: relative;
    border: 1px solid black;
    background: #eee;
  
    &:hover {
      background: #ff9898;
      transform: translate(0, -20%);
      transition: 0.5s;
    }
  }
}
```